src/particles.zig

```diff
@@ -60,12 +65,22 @@ pub const ParticleManager = struct {
 		};
 
 		const particleType = readTextureDataAndParticleType(assetsFolder, textureId);
+		var rotVel: RandomRange(f32) = RandomRange(f32).fromZon(zon.getChild("rotationVelocity")) orelse .init(20, 60);
+		rotVel.min = std.math.pi/180.0;
+		rotVel.max = std.math.pi/180.0;
```

Comment:
This does not seem right. Did you mean to multiply?
Also I would suggest to use std.math.degreesToRadians instead to make it more explicit.